kong/types.go

```diff
@@ -157,7 +157,7 @@ type PassiveHealthcheck struct {
 type Healthcheck struct {
 	Active    *ActiveHealthcheck  `json:"active,omitempty" yaml:"active,omitempty"`
 	Passive   *PassiveHealthcheck `json:"passive,omitempty" yaml:"passive,omitempty"`
-	Threshold *float64            `json:"threshold,omitempty" yaml:"threshold,omitempty"`
+	Threshold *json.Number        `json:"threshold,omitempty" yaml:"threshold,omitempty"`
```

Comment:
Has it been verified that this change fixes the `make manifests` error in railgun?